napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -131,20 +131,60 @@ def _get_current_tooltip_visibility() -> bool:
 
 MENUID_DICT = {'axes': MenuId.VIEW_AXES, 'scale_bar': MenuId.VIEW_SCALEBAR}
 
-for cmd, viewer_attr, sub_attr in (
-    (CommandId.TOGGLE_VIEWER_AXES, 'axes', 'visible'),
-    (CommandId.TOGGLE_VIEWER_AXES_COLORED, 'axes', 'colored'),
-    (CommandId.TOGGLE_VIEWER_AXES_LABELS, 'axes', 'labels'),
-    (CommandId.TOGGLE_VIEWER_AXES_DASHED, 'axes', 'dashed'),
-    (CommandId.TOGGLE_VIEWER_AXES_ARROWS, 'axes', 'arrows'),
-    (CommandId.TOGGLE_VIEWER_SCALE_BAR, 'scale_bar', 'visible'),
-    (CommandId.TOGGLE_VIEWER_SCALE_BAR_COLORED, 'scale_bar', 'colored'),
-    (CommandId.TOGGLE_VIEWER_SCALE_BAR_TICKS, 'scale_bar', 'ticks'),
+for cmd, cmd_title, viewer_attr, sub_attr in (
```

Comment:
The start of the `for` is now separated from the body of the loop by a huge gap now, which makes this really hard to read. I suggest one or both of:
- add a comment above the for loop that says what the for-loop is testing, and/or
- make the list of tuples before the loop, then have the loop iterate over that list.

Reply:
Good idea. Will do.

Reply:
@jni done, thank you!

Reply:
Ah forgot to pull before pushing. Done @jni !